.clang-format

```diff
@@ -3,7 +3,7 @@ Language:        Cpp
 # BasedOnStyle:  Chromium
 AccessModifierOffset: -1
 AlignAfterOpenBracket: Align
-AlignConsecutiveAssignments: true
+AlignConsecutiveAssignments: false
```

Comment:
Agree on this part, since it conflicts with PointerAlignment which I believe should take precedence. Does this mean AlignConsecutiveDeclarations should also be false?